autoload/leaderf/python/leaderf/mruExpl.py

```diff
@@ -105,6 +130,8 @@ def _argaddFiles(self, files):
         # It will raise E480 without 'silent!'
         lfCmd("silent! argdelete *")
         for file in files:
+            if lfEval("get(g:, 'Lf_ShowDevIcons', 1)") == "1":
+                file = file[file.find(' ')+1:]
```

Comment:
What are these two lines used for? I think they should be removed.

Reply:
Thanks.
I fixed.